_studio/mfx_lib/encode_hw/hevc/linux/base/hevcehw_base_va_lin.cpp

```diff
@@ -191,6 +195,17 @@ void DDI_VA::InitAlloc(const FeatureBlocks& /*blocks*/, TPushIA Push)
         MFX_CHECK((attrib[1].value & vaRCType), MFX_ERR_DEVICE_FAILED);
 
         attrib[1].value = vaRCType;
+        priority = core.GetSession()->m_priority;
+        if (priority == MFX_PRIORITY_LOW)
+        {
+            attrib[2].value = -512;
```

Comment:
-512/512 should go via dymanic caps from UMD or some harcoded value from va.h

Reply:
Yep, no magic values

Reply:
Depends on the VAAPI change (https://github.com/intel/libva/pull/416) final decision now.  Once the VAAPI change is merged, it will be updated then.

Reply:
I agree. We need to get these values from somewhere, either having them declared in VAAPI or VAAPI returning a supported range sorted be priority in a API declared way. In this case we can do calculation of min, (min+max)/2, max and map that to LOW, NORMAL, HIGH.

Reply:
Updated. The max value is reported from media driver. LOW is map to 0, NORMAL is map to max_value/2; HIGH is map to max value.